### DISCUSSION TITLE
EfficientNetv2 in Flax

### DISCUSSION THREAD

**Discussion by faresobeid** [google/flax#3378] [Q&A]
I wanted to use a pre-trained efficientnetv2 model in flax but have found no easy way to do this. I'm using kaggle and want to import the pre-trained model and finetune it. What is the easiest way to do this?

**Comment by chiamp** [comment]
Just to clarify, are you trying to import a pre-trained efficientnetv2 model trained in Flax onto your Kaggle environment? Where is the source of this pre-trained model?
